sinatra-contrib/sinatra-contrib.gemspec

```diff
@@ -23,7 +23,8 @@ Gem::Specification.new do |s|
 
   s.add_dependency "sinatra", version
   s.add_dependency "mustermann", "~> 1.0"
-  s.add_dependency "backports", "~> 2.8.2"
+  s.add_dependency "backports", ">= 2.8.2"
+  s.add_dependency "activesupport", ">= 4.0.0"
```

Comment:
Just to spare anyone else from digging, here's the relevant minimum gemspec: https://github.com/rails/rails/blob/4-0-stable/activesupport/activesupport.gemspec

But I guess Bundler will try to use a much newer version than that for most of us